docs/serializer-formatter.rst

```diff
@@ -83,25 +83,29 @@ not well formed.
     import kafka.consumer.ConsumerIterator;
     import kafka.consumer.KafkaStream;
     import kafka.javaapi.consumer.ConsumerConnector;
-    import java.util.Properties;
+    import io.confluent.kafka.serializers.KafkaAvroDecoder;
+    import kafka.message.MessageAndMetadata;
+    import kafka.utils.VerifiableProperties;
+    import org.apache.kafka.common.errors.SerializationException;
+    import java.util.*;
 
     Properties props = new Properties();
     props.put("zookeeper.connect", "localhost:2181");
     props.put("group.id", "group1");
     props.put("schema.registry.url", "http://localhost:8081");
 
     Map<String, Integer> topicCountMap = new HashMap<String, Integer>();
-    topicCountMap.put(topic, new Integer(1));
+    topicCountMap.put("topic1", new Integer(1));
 
     VerifiableProperties vProps = new VerifiableProperties(props);
     KafkaAvroDecoder keyDecoder = new KafkaAvroDecoder(vProps);
     KafkaAvroDecoder valueDecoder = new KafkaAvroDecoder(vProps);
 
-    kafka.consumer.Consumer.createJavaConsumerConnector(new ConsumerConfig(vProps));
+    ConsumerConnector consumer = kafka.consumer.Consumer.createJavaConsumerConnector(new ConsumerConfig(props));
```

Comment:
From the `ConsumerConfig` code, it looks like you can actually instantiate with `vProps`, and it would be more consistent with the other usages?